locallib.php

```diff
@@ -87,12 +87,17 @@
  *
  * @return string
  */
-function bigbluebuttonbn_get_join_url($meetingid, $username, $pw, $logouturl, $configtoken = null, $userid = null) {
+function bigbluebuttonbn_get_join_url($meetingid, $username, $pw, $logouturl, $configtoken = null, $userid = null, $clienttype=0) {
```

Comment:
For the default value and the comparisions, I would use a constant to make it more explicit.